docs/swagger.yaml

```diff
@@ -341,6 +469,61 @@ components:
           example: 'https://example.com/myProduct'
       required:
         - redirectUrl
+    MerchantCallbackPutRequest:
+      title: MerchantCallbackPutRequest
+      x-stoplight:
+        id: 56a28c70438c7
+      type: object
+      x-examples:
+        Example 1:
+          location: Kasse 1
+      properties:
+        location:
+          type: string
+          description: 'A description of where the QR code will be located. It will be shown in the app when a user scans the QR code. Examples could be ‘Kasse 1’ , ‘Kiosk’ or ‘Platform 3’.'
+      required:
+        - location
+    MerchantCallbackMobilePayPutRequest:
+      title: MerchantCallbackMobilePayPutRequest
+      type: object
+      x-examples:
+        Example 1:
+          location: Kasse 1
+      properties:
+        location:
+          type: string
+          description: 'A description of where the QR code will be located. It will be shown in the app when a user scans the QR code. Examples could be ‘Kasse 1’ , ‘Kiosk’ or ‘Platform 3’.'
+      required:
+        - location
+      x-stoplight:
+        id: 574e5e977f056
+    MerchantCallbackQr:
+      title: MerchantCallbackQr
+      x-stoplight:
+        id: a8x4nog6fas22
+      type: object
+      description: 'A representation of a QR code. It contains all the three properties that was provided when creating the QR code, in addition to a link pointing to the actual QR code itself.'
+      x-examples:
+        Example 1:
+          merchantSerialNumber: '12345'
```

Comment:
Not sure if there is any better data for this, but we a RFC for examples here:
https://github.com/vippsas/rfc/blob/master/docs/0001-api-guidelines.md#examples-of-data-types-and-values

Also a lot of good info about how our external api should look :)

Reply:
Do you think I should change the name of merchantSerialNumber to 'msn' as the example says? - Also the example for msn is '123456' but other examples in the same document it is '12345'. The MSN is always five-digit long right?

Reply:
I appreciate the link to the document :) Its nice there are guidelines as those for the whole organisation. That is good practice to align all teams on this :)